src/microbit/pxt_snake/snake_nn.md

```diff
@@ -0,0 +1,651 @@
+		drawSnake();
+		updateSnake();
+		basic.pause(1000);
+	})
+	```
+
+# Steg 3: Styring {.activity}
+
+No er det på tide å leggje inn styring av slangen. Men fyrst lagar me ein liten
+startskjerm.
+
+## Steg for steg {.check}
+
+- [ ] Lag ein ny variabel. Kall variabelen `isPlaying` og set den til `false` i
+  startblokka. Aller fyrst i startblokka legg du ein `show icon`-kloss med
+  "snake"-ikonet.
+
+![Bilete av start](show_startscreen.png)
+
+- [ ] I `forever`-funksjonen legg du ein `if` som testar om `isPlaying` er sann
+  (`true`). I så fall skal programmet teikne og oppdatere, så dette puttar du
+  inni `if`-klossen.
+
+![Bilete av forever](snake_mainloop_steg2.png)
+
+- [ ] Så legg me til funksjonar for knappane. Når knappen vert trykt set du
+  `isPlaying` til `true`
+
+![Bilete av knappekode for å starte](Buttons_code_start.png)
+
+## Test prosjektet i simulatoren {.flag}
+
+No skal slangeikonet visast til du trykkar på ein knapp, då startar spelet. Men
+me treng meir kode for å styre. Når ein speler skal A-knappen styre slangen 90
+grader mot venstre frå noverande retning, medan B-knappen styrer slangen 90
+grader mot høgre.
+
+## Steg for steg {.check}
+
+- [ ] Legg til ein `if-else` i knappekoden. Viss me er i `isPlaying`-modus, så
+  skal knappetrykket styre slangen. Viss me trykkar på A-knappen og `direction`
+  er `up`, så endrar me den til `left`. Viss den er `left` endrar me til `down`,
+  osb. Og gjer motsett for B-knappen. Viss me ikkje er i `isPlaying`-modus, så
+  må me skru den på.
+
+```Javascript
+input.onButtonPressed(Button.A, function () {
+    if (isPlaying) {
+        if (direction == 'up') {
+        	direction = 'left';
+        }
+        else if (direction == 'left') {
+            direction = 'down';
+        }
+        else if (direction == 'down') {
+            direction = 'right';
+        }
+        else if (direction == 'right') {
+            direction = 'up';
+        }
+    } else {
+        isPlaying = true
+    }
+})
+input.onButtonPressed(Button.B, function () {
+    if (isPlaying) {
+        if (direction == 'up') {
+            direction = 'right';
+        }
+        else if (direction == 'right') {
+            direction = 'down';
+        }
+        else if (direction == 'down') {
+            direction = 'left';
+        }
+        else if (direction == 'left') {
+            direction = 'up';
+        }
+    } else {
+        isPlaying = true;
+    }
+})
+```
+
+## Test prosjektet {.flag}
+
+__Prøv spelet i simulatoren for å teste koden så langt. Sjekk at styringa
+virkar.__
+
+No kan du styre slangen, men det er eit lite problem. Viss me trykkar fort to
+gonger på ein knapp går slangen i motsett retning. Det vil me ikkje, sidan det
+tyder at slangen går gjennom seg sjølv, og det blir berre tull.
+
+## Steg for steg {.check}
+
+- [ ] Legg til ein ny variabel som heiter `buttonPressed`.
+
+- [ ] Endre begge `onButtonPressed`-funksjonane med følgjande kode:
+
+```Javascript
```

Comment:
her må vi kanskje se på JS syntax i begge oppgaver. vi må sørge for at alle `{` lukkes skikkelig, samt at de er "alignet".

slik ser det ut på https://oppgaver.kidsakoder.no/microbit/pxt_snake/snake:
![image](https://user-images.githubusercontent.com/5648743/61710150-35ea1400-ad51-11e9-8233-314fe2db3912.png)

i nynorskversjonen ser det slik ut:
![image](https://user-images.githubusercontent.com/5648743/61710174-47cbb700-ad51-11e9-8c16-d63792308263.png)